src/PowerShellEditorServices/Session/RemoteFileManager.cs

```diff
@@ -47,7 +47,17 @@ public class RemoteFileManager
                     $filePathName = $_.FullName
 
                     # Get file contents
-                    $contentBytes = Get-Content -Path $filePathName -Raw -Encoding Byte
+                    $params = @{ Path=$filePathName; Raw=$true }
+                    if ($PSVersionTable.PSEdition -eq 'Core')
+                    {
+                        $params += @{ AsByteStream=$true }
```

Comment:
It should be a bit more performant (and PS canonical IMO) to do this: `$params['AsByteStream']=$true`.  This way we modify the original hashtable without creating a new to merge with the original one.

Reply:
updated 🙂